nva-datamodel-java/src/test/java/no/unit/nva/PublicationContextTest.java

```diff
@@ -72,7 +72,7 @@ void shouldWriteEveryNvaClassAsFragmentUri() {
     void shouldDefineEveryArrayPropertyAsContainerSet() {
         var allNvaCollectionProperties = generateSetOfPropertiesThatHaveCollectionTypeForEveryNvaType();
         var contextContainerSets = getAllContextContainerSetTerms();
-        assertThat(contextContainerSets, is(equalTo(allNvaCollectionProperties)));
+        assertThat(allNvaCollectionProperties, is(equalTo(contextContainerSets)));
```

Comment:
Hva er grunnen til at du endret rekkefølgen? Jeg er bare nysgjerrig :)

Reply:
Såvidt jeg kunne se er allNvaCollectionProperties expected og contextContainerSets actual, var bare for at feilmeldingen gir mer mening :)